pydicom/tests/test_filereader.py

```diff
@@ -114,6 +115,11 @@ def test_UTF8_filename(self):
         os.remove(utf8_filename)
         assert ds is not None
 
+    def test_pathlib_path_filename(self):
+        """Check that file can be read using pathlib.Path"""
+        ds = dcmread(Path(priv_SQ_name))
+        assert ds is not None
+
```

Comment:
Nitpick, but is it possible to be None? There should be an error on previous line if something happened so is the assert line needed?

Reply:
Yes - that was copy and paste, I will remove it.

Reply:
Ok, done.